packages/phrases/src/locales/en/errors.ts

```diff
@@ -7,6 +7,7 @@ const errors = {
     expected_role_not_found:
       'Expected role not found. Please check your user roles and permissions.',
     jwt_sub_missing: 'Missing `sub` in JWT.',
+    require_re_authentication: 'Re-authentication is required to perform a protected action.',
```

Comment:
Should we also mark this as `// UNTRANSLATED`

Reply:
There is no need to translate `en`?

Reply:
Ah, my bad